src/services/Hooks/useCheckPermissions.ts

```diff
@@ -0,0 +1,41 @@
+import * as Notifications from 'expo-notifications';
```

Comment:
this runs when reaching calendar screen, will ask for permission and requests expo token from expo. It also updates the db with the expo token